group_vars/rabbitmq/public.yml

```diff
@@ -6,5 +6,9 @@ SANITY_CHECK_LIVE_PORTS:
     port: 15671
     message: "RabbitMQ management does not respond on port 15671."
 
+SANITY_CHECK_COMMANDS:
+  - command: /usr/sbin/rabbitmqctl node_health_check | grep 'Health check passed'
```

Comment:
Any reason for the `grep`?  The return code of `rabbitmqctl` seems already correct.

Reply:
The [documentation](https://www.rabbitmq.com/rabbitmqctl.8.html#node_health_check) doesn't specify the return code. The command could write some warning (like „some tests passed, others not“) and still return 0; probably not, but I'd need to check the source code to be sure, therefore better to use something we control.
I can change it if we trust it.

Reply:
Relying on the exact textual output seems a lot more fragile than relying on the return code to me.  I tried that the return code is non-zero when the memory alert is active, so at least the error condition we know would be covered.  So I'd ask you to remove the `grep` here.  Once you have done this, you can deploy this to `rabbitmq-stage`.

Reply:
Being fragile is a feature for a health checker :-) But ok, I removed it and will merge it and deploy carefully to stage.